electron/main.ts

```diff
@@ -7,6 +7,23 @@ import store from './services/config-store'
 import { setupNetworkService } from './services/network'
 import { setupFilesystemStorage } from './services/storage'
 
+// Enable hardware acceleration
+console.log('Electron GPU info:')
+console.log(app.getGPUFeatureStatus())
+app.getGPUInfo('complete').then((info) => {
+  console.log(info)
+})
+
+app.commandLine.appendSwitch('use-gl', 'egl')
+app.commandLine.appendSwitch('ignore-gpu-blocklist')
+app.commandLine.appendSwitch('enable-accelerated-video-decode')
+app.commandLine.appendSwitch('enable-zero-copy')
+app.commandLine.appendSwitch('enable-gpu-rasterization')
+app.commandLine.appendSwitch('enable-oop-rasterization')
+app.commandLine.appendSwitch('enable-pinch')
+app.commandLine.appendSwitch('flag-switches-begin')
+app.commandLine.appendSwitch('flag-switches-end')
```

Comment:
From some documentation:
> These two flags are added **around** the switches chrome:flags adds to the command line. This is useful to see which switches were added by chrome:flags on chrome:version. They don't have any effect.

Since there is nothing around this flags, it does not make sense to have it here.

Reply:
from source code:
https://chromium.googlesource.com/chromium/src/+/c1a515dfc10c025e5406f19b51f47ecba5496f5b/components/flags_ui/flags_state.cc#44

Reply:
Yes yes. This is a first test literally replicating the same arguments that the chromium version that Tony was running had, but those will be filtered.